deploy.sh

```diff
@@ -1,6 +1,5 @@
 #!/bin/bash
 # Deploys to GitHub Pages.
-# Make sure to stash your local changes before running this script.
```

Comment:
There's a npm postinstall script that revert some files. I need to fix that but in the meantime, please revert deploy.sh and gulpfile.js or don't commit the changes.